roles/ocp_on_libvirt/tasks/libvirt_host_destroy.yml

```diff
@@ -1,20 +1,31 @@
 ---
-- name: "Does node exist"
+- name: "Does node exist - {{ host['name'] }}"
```

Comment:
I think this task can be generalized and executed from https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/roles/ocp_on_libvirt/tasks/main.yml, probably as a first task that can be used for both install and cleanup hook_action, because the output is exactly the same on every iteration for both cases and the variable name is the same.
I would also change the task name to something more representative, such as `List VMs`

Reply:
You will get different list each iteration you will create new node or destroy one, but we can take this task outside

Reply:
In the next PR I'll create loop on "hosts" with no need to get single node from outside